src/openrct2/actions/FootpathPlaceAction.hpp

```diff
@@ -182,16 +183,31 @@ DEFINE_GAME_ACTION(FootpathPlaceAction, GAME_COMMAND_PLACE_PATH, GameActionResul
         }
 
         pathElement->SetSurfaceEntryIndex(_type & ~FOOTPATH_ELEMENT_INSERT_QUEUE);
-        if (_type & FOOTPATH_ELEMENT_INSERT_QUEUE)
-        {
-            pathElement->SetIsQueue(true);
-        }
-        else
+        bool isQueue = _type & FOOTPATH_ELEMENT_INSERT_QUEUE;
+        pathElement->SetIsQueue(isQueue);
+
+        rct_scenery_entry* elem = pathElement->GetAdditionEntry();
+        if (elem != nullptr)
         {
-            pathElement->SetIsQueue(false);
+            if (isQueue)
+            {
+                // remove any addition that isn't a TV or a lamp
+                if ((elem->path_bit.flags & PATH_BIT_FLAG_IS_QUEUE_SCREEN) == 0 && (elem->path_bit.flags & PATH_BIT_FLAG_LAMP) == 0)
```

Comment:
if ((elem->path_bit.flags & PATH_BIT_FLAG_IS_QUEUE_SCREEN) == 0
+                    && (elem->path_bit.flags & PATH_BIT_FLAG_LAMP) == 0)
```suggestion
                if ((elem->path_bit.flags & PATH_BIT_FLAG_IS_QUEUE_SCREEN) == 0
                    && (elem->path_bit.flags & PATH_BIT_FLAG_LAMP) == 0)
```
Just to fix formatting for everything to be ok, CI-wise